app/src/main/java/com/kroger/start/MainActivity.kt

```diff
@@ -12,6 +15,14 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        // Compose
+        setContent {
+            MaterialTheme {
+                ComicScreen()
+            }
+        }
+
+        // Fragment/XML
```

Comment:
Should we comment one or both of these? What happens UI-wise when both are trying to render?

Reply:
I didn't comment either out on purpose figuring the interviewer could explain when they get to that file, but after sitting with it for a bit, I think we could comment out the fragment/xml part. Most folks these days are using compose, so it feels like that's what we should make more accessible, while still giving the option for doing it the old way. Also we want people to use compose, so a little nudge in that direction feels fine.

Reply:
To answer the question though, when setContent is called twice, the second one just overrides the first. doesn't seem to cause any issues